src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -46,21 +46,21 @@
         public static final double ARM_REACH_DISTANCE = Math.PI * RUNG_DISTANCE * GEARBOX_REDUCTION
                 * ENCODER_TICKS_PER_REVOLUTION;
 
-        public static final double ENCODER_TICKS_PER_INCH = 78417 / 11.5;
+        public static final double ENCODER_TICKS_PER_INCH = 78417 / 11.5 * 2;
 
         public static final double P = 0.5;
         public static final double I = 0;
         public static final double D = 0;
 
         public static final int PID_SLOT_0 = 0;
 
-        public static final double MID_RUNG_HEIGHT_INCH = 30;
-        public static final double RETRACT_HEIGHT_INCH = 2;
+        public static final double MID_RUNG_HEIGHT_INCH = 31;
+        public static final double RETRACT_HEIGHT_INCH = 20;
 
         public static final double CLIMB_OFFSET_INCHES = 28.5;
 
         public static final SupplyCurrentLimitConfiguration MOTOR_CURRENT_LIMIT = new SupplyCurrentLimitConfiguration(
-                true, 40, 40, 500);
+                true, 60, 60, 1000);
```

Comment:
60, 60?  Aren't the breakers and the PDH limited to a maximum of 40 Amps?

Reply:
The breakers are able sustain 40amps indefinitely but can go above it temporarily. I've been told 150% for 15 seconds is allowed

Reply:
But it doesn't seem like this ever limits the motor back down to 40a, even after the limit triggers. 
Shouldn't this be along the lines of 40, 60, 15?

Reply:
Yeah this seems a bit off, especially considering that the time units are in seconds, and 1000 seconds is longer than an entire match.